Binance.Net/BinanceFuturesSocketClient.cs

```diff
@@ -0,0 +1,744 @@
+using Binance.Net.Converters;
+using Binance.Net.Objects;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Binance.Net.Interfaces;
+using Binance.Net.Objects.Sockets;
+using CryptoExchange.Net;
+using CryptoExchange.Net.Authentication;
+using CryptoExchange.Net.Logging;
+using CryptoExchange.Net.Objects;
+using CryptoExchange.Net.Sockets;
+using Newtonsoft.Json.Linq;
+
+namespace Binance.Net
+{
+    /// <summary>
+    /// Client providing access to the Binance websocket Api
+    /// </summary>
+    public class BinanceFuturesSocketClient : SocketClient, IBinanceFuturesSocketClient
+    {
+        #region fields
+        private static BinanceFuturesSocketClientOptions defaultOptions = new BinanceFuturesSocketClientOptions();
+        private static BinanceFuturesSocketClientOptions DefaultOptions => defaultOptions.Copy();
+
+        private readonly string baseCombinedAddress;
+
+        private const string AggregatedTradesStreamEndpoint = "@aggTrade";
+        private const string MarkPriceStreamEndpoint = "@markPrice";//TODO
```

Comment:
These TODO's still have to be implemented?

Reply:
I had put this down as a note to myself not to forget. These are complete now